projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/AccessLevel.java

```diff
@@ -0,0 +1,23 @@
+package org.batfish.datamodel.pojo;
+
+import org.batfish.common.BatfishException;
+
+public enum AccessLevel {
```

Comment:
This enum class will be used to determine how much information about a pojo object will be returned. e.g. function calls will be in form of: getContainer(String containerName, AccessLevel accessLevel), etc.